services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseTestAbstractBase.java

```diff
@@ -74,7 +78,7 @@ public void tearDown() throws Exception {
    }
 
    @Nullable protected UserDbInterfaceImpl bindToDbService() {
-      Context context = InstrumentationRegistry.getContext();
+      Context context = ApplicationProvider.getApplicationContext();
```

Comment:
I believe you are getting the wrong context here. The test used to get the "instrumentation/testing" context not the application under test.